libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -201,7 +201,7 @@ void AP_RPM::update(void)
 
             drivers[i]->update();
 
-#if HAL_WITH_ESC_TELEM
+#if AP_RPM_ESC_TELEM_ENABLED
```

Comment:
Current code is correct. It should be able to run with or without ```AP_RPM_ESC_TELEM_ENABLED```. In fact, this only runs if type is **NOT** RPM_TYPE_ESC_TELEM

Reply:
Reverted this patch